### PR TITLE
Release 1.7.5

### PR DESCRIPTION
**Pull request by rail**

### CODE REVIEW

File: deploy/certified-metadata-bundle/cockroach-operator/1.7.5/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml [hunk 1]
```diff
@@ -275,51 +275,53 @@ spec:
...
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cockroachdb
+                - name: RELATED_IMAGE_COCKROACH_v19_2_4
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/1.7.5/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml:322]: Same issue as above

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.5.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cockroachdb
+                - name: RELATED_IMAGE_COCKROACH_v19_2_4
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.5.clusterserviceversion.yaml:322]: same issue as above

File: deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cockroachdb
+                - name: RELATED_IMAGE_COCKROACH_v19_2_4
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml:322]: Same issue as above

**Review by keith-mcclellan — CHANGES_REQUESTED** [review]

File: deploy/certified-metadata-bundle/cockroach-operator/1.7.5/cockroach-operator.v1.7.5.clusterserviceversion.yaml [hunk 4]
```diff
@@ -275,51 +275,53 @@ spec:
...
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cockroachdb
+                - name: RELATED_IMAGE_COCKROACH_v19_2_4
```
[the remark this answers is not in the document]
Comment (rail) [reply deploy/certified-metadata-bundle/cockroach-operator/1.7.5/cockroach-operator.v1.7.5.clusterserviceversion.yaml]: Something went wrong with my initial addition of all supported images.

File: deploy/certified-metadata-bundle/cockroach-operator/1.7.5/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml (hunk 1 above)
Comment (rail) [reply deploy/certified-metadata-bundle/cockroach-operator/1.7.5/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml]: Fixed

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.5.clusterserviceversion.yaml (hunk 2 above)
Comment (rail) [reply deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.5.clusterserviceversion.yaml]: Fixed

File: deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml (hunk 3 above)
Comment (rail) [reply deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/cockroach-operator.v1.7.5.clusterserviceversion.yaml]: Fixed

File: manifests/operator.yaml [hunk 5]
```diff
@@ -285,7 +285,7 @@ spec:
               value: cockroachdb
             - name: RELATED_IMAGE_COCKROACH_v19_2_12
```
Comment (keith-mcclellan) [inline manifests/operator.yaml:286]: This looks wrong to me still

**Review by keith-mcclellan — CHANGES_REQUESTED** [review]
manifests/operator.yaml doesn't reflect the same changes as the CSV

**Review by keith-mcclellan — APPROVED** [review]
LGTM

**Pull request merged**